helm-eshell.el

```diff
@@ -71,6 +77,7 @@
    (nomark :initform t)
    (persistent-action :initform 'ignore)
    (nohighlight :initform t)
+   (fuzzy-match :initform (eval helm-eshell-fuzzy-match))
```

Comment:
bb2020 <notifications@github.com> writes:

> In helm-eshell.el:
>
>> @@ -71,6 +77,7 @@
>>     (nomark :initform t)
>>     (persistent-action :initform 'ignore)
>>     (nohighlight :initform t)
>> +   (fuzzy-match :initform (eval helm-eshell-fuzzy-match))
>
> I think it is better to run :before. Otherwise it breaks classical helm completion.

And did you try adding:

(fuzzy-match :initform helm-eshell-fuzzy-match)

I think eval isn't needed.

> (defmethod helm--setup-source :before ((source helm-esh-source))
>   (setf (slot-value source 'fuzzy-match) helm-eshell-fuzzy-match))
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub, or mute the thread.*


-- 
Thierry